frontend/src/citizen-frontend/App.tsx

```diff
@@ -67,10 +68,17 @@ export default function App() {
                     {featureFlags.experimental?.incomeStatements && (
                       <Route
                         exact
-                        path="/income/:incomeStatementId"
+                        path="/income/:incomeStatementId/edit"
                         component={requireAuth(IncomeStatementEditor)}
                       />
                     )}
+                    {featureFlags.experimental?.incomeStatements && (
+                      <Route
+                        exact
+                        path="/income/:incomeStatementId/view"
```

Comment:
Any reason to use the `/view` suffix instead of just:
```suggestion
                        path="/income/:incomeStatementId"
```
?

Looks like the pattern so far has mostly been without the suffix for "view-views".

Reply:
https://github.com/espoon-voltti/evaka/pull/1332/commits/38f409c5389ea2ce6b74e0ca5183c4f10a70e2d1